Source/Core/VideoBackends/D3D12/Render.cpp

```diff
@@ -1244,6 +1242,8 @@ void Renderer::SetSamplerState(int stage, int tex_index, bool custom_tex)
 	if (tex_index)
 		stage += 4;
 
+	const auto original_hex = gx_state.sampler[stage].hex;
```

Comment:
@lioncash Okay, [that was my original thought](https://gist.github.com/EmptyChaos/033ea74045c6b818534b), but doesn't that still have packing problems when you aggregate those classes? i.e. isn't it still possible for IND_MTX to be padded between its IND_MTXA/IND_MTXB... member variables, or for any of the classes to have trailing padding at the end of each class (e.g. a platform with 64bit alignment would pad a u32 to 8bytes)? I haven't been able to find any references that say `sizeof(class) == sizeof(only_member)` or `alignof(class) == alignof(only_member)` without using the `#pragma pack` extension, most information seems to explicitly claim the opposite.